service/src/main/java/bio/terra/pipelines/db/entities/PipelineRun.java

```diff
@@ -37,9 +37,15 @@ public class PipelineRun {
   @Column(name = "pipeline_id", nullable = false)
   private Long pipelineId;
 
-  @Column(name = "workspace_id", nullable = false)
+  @Column(name = "workspace_id")
   private UUID workspaceId;
 
+  @Column(name = "workspace_project")
```

Comment:
nitpicky but these could be updated to be `nullable = false` for now but would be removed later if we went back to adding workspaceId hahaha so nvm :)